src/index.ts

```diff
@@ -301,7 +307,10 @@ async function getBalance() {
 
   const ledger = LedgerCanister.create({
     agent: await getCurrentAgent(new AnonymousIdentity()),
-    hardwareWallet: true,
+    hardwareWallet: await isCurrentVersionSmallerThan({
```

Comment:
```
await isCurrentVersionSmallerThan({
      identity,
      version: FULL_CANDID_PARSER_VERSION,
    })
```

is repeated few times, worth a util function?

Reply:
Makes sense, sure.